pkg/controller/cloud/node_controller.go

```diff
@@ -192,7 +192,7 @@ func (cnc *CloudNodeController) updateNodeAddress(node *v1.Node, instances cloud
 	}
 }
 
-func (cnc *CloudNodeController) UpdateCloudNode(_, newObj interface{}) {
+func (cnc *CloudNodeController) UpdateCloudNode(ctx context.Context, _, newObj interface{}) {
```

Comment:
Is there value in passing context into the Add/Update event handlers if client-go informers don't pass in a context anyways? Wondering if the context should start inside AddCloudNode and UpdateCloudNode instead.

Reply:
Great question. 

There has been discussion in SIG api machinery about adding context to client-go. So eventually we should have a context there. I think we have a lot of places this needs to be fixed and I would like to do it in small steady chunks. This will also help from the other direction. There has been talk about trying to get controller managers able to shut down gracefully. Part of that is the ability to pass through a cancel/done message. Once this has been fully piped through it will help with that as well. I agree there is little immediate value to this change. That is because the change has to be made in a lot of places to be helpful. However to get that to happen we need to make a lot of these small changes.

Reply:
gotcha, thanks for the context (pun intended)